openmetadata-ui/src/main/resources/ui/src/utils/Assets/AssetsUtils.ts

```diff
@@ -66,3 +68,14 @@ export const getEntityAPIfromSource = (
       return getContainerByName;
   }
 };
+
+export const handleDataAssetAfterDeleteAction = (isSoftDelete?: boolean) => {
+  if (isSoftDelete) {
+    setTimeout(() => {
+      refreshPage();
```

Comment:
You can directly use `history.go(0)` here.

Reply:
pushed the changes